src/sentry/api/base.py

```diff
@@ -1,12 +1,10 @@
 from __future__ import absolute_import
 
-__all__ = ['DocSection', 'Endpoint', 'StatsMixin']
-
 import logging
-import six
 import time
-
 from datetime import datetime, timedelta
+
+import six
```

Comment:
I didn't (intentionally, at least.) This is using the `isort` configuration you wrote: 42522f5025fd25749ba7911939ad53f83611e627

😕